lib/commands/install.js

```diff
@@ -44,6 +45,9 @@ class InstallCommand extends Command {
         }, argv, {local})).then(() => this.ui.listr([{
             title: 'Checking for latest Ghost version',
             task: this.version
+        }, {
```

Comment:
does it make sense to show release notes on install? this may add a lot of unnecessary output for someone who's just getting set up for the first time

Reply:
True! Removing...